robolectric/src/test/resources/res/values/typed_arrays.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <array name="typed_array_values">
+        <item>abcdefg</item>
+        <item>3875</item>
+        <item>2.0</item>
+        <item>#ffff00ff</item>
+        <item>#00ffff</item>
+    </array>
+    <array name="typed_array_references">
+        <item>@string/typed_array_a</item>
```

Comment:
Some of those types aren't accessible via TypedArrays, but I've added the others.